src/docs/Foundations/Colors/index.mdx

```diff
@@ -52,36 +52,36 @@ We used the same principles to design our greyscale but greys are based on **11
 
 # Status colors
 
-Status colors convey various levels of importance for feedback informations, like errors or warning. They can also highlight important informations or tips for the user or inform him that an operation had fail of succeded.
+Status colors convey various levels of importance for feedback information, like errors or warnings. They can also highlight important pieces of information or some tips. Status colors can also inform the user that an operation has failed or succeeded.
 
-## Info
+## Information
 
-Used to highlight important information the user should be aware about.
+This color is used to highlight important pieces of information the user should be aware of.
 Should only be used if the information does not represent a threat.
 
 <Swatch id="info" />
 
 ## Success
 
-Inform the user that an operation was successful.
+Informs the user that an operation was successful.
 
 <Swatch id="success" />
 
-## Warning
+## Warnings
 
-Used to inform abouts risks or threats. It teel the user to be carefull about something.
+Inform the user about risks or threats. It teels the user to be careful about something.
 
 <Swatch id="warning" />
 
 ## Danger
 
-Used to inform user of an error, or that some operation whent wrong.
+Inform the user of an error or that some operation went wrong.
```

Comment:
This color informs the user of an error or that some operation went wrong. 

(I think it's better for the global understanding 😉)